src/coreclr/inc/cor.h

```diff
@@ -2233,7 +2233,11 @@ inline ULONG CorSigCompressPointer( // return number of bytes of that compressed
     void * pvPointer,               // [IN] given uncompressed data
     void * pData)                   // [OUT] buffer where iLen will be compressed and stored.
 {
+#ifdef TARGET_LOONGARCH64
```

Comment:
Changes like this should not be under. This change should be done for all architectures.

Reply:
Yes, you are right.
Especially for CPU processors supporting unaligned memory accessing.
I will update it later with breaking up this PR into serval PRs.